tests/unit/composables/useAlert.spec.js

```diff
@@ -1,13 +1,25 @@
 import useAlert from '@/composables/useAlert.js'
 import store from '@/store'
+import { useRouter } from 'vue-router'
```

Comment:
We have a router defined in support/testHelper.js. Wondering if thats the issue here, have you tried importing that and stubbing it.